.github/workflows/pr-contributor-welcome.yml

```
@@ -19,7 +19,15 @@ jobs:
     if: github.event.pull_request.merged == true && github.repository == 'ant-design/ant-design'
     runs-on: ubuntu-latest
     steps:
+      - name: get commit count
+        id: get_commit_count
+        run: |
+          PR_AUTHOR=$(echo "${{ github.event.pull_request.user.login }}")
+          RESULT_DATA=$(curl -s "https://api.github.com/repos/${{ github.repository }}/commits?author=${PR_AUTHOR}&per_page=5")
+          DATA_LENGTH=$(echo $RESULT_DATA | jq 'if type == "array" then length else 0 end')
+          echo "COUNT=$DATA_LENGTH" >> $GITHUB_OUTPUT
       - name: Comment on PR
+        if: steps.get_commit_count.outputs.COUNT < 2
```

要不条件改宽一点，过去有很多人贡献过 antd。
```suggestion
        if: steps.get_commit_count.outputs.COUNT < 3
```

---

空投风波有很多恶意蹭pr的人在，但是感觉都是无头小号，人家也不是奔着群来的，而且还有一个审核群在可以看pr质量把控，会不会卡住一些对开源感兴趣的新人(其实看看账号活跃度应该能排挤掉坏人）。

---

> 要不条件改宽一点，过去有很多人贡献过 antd。

哈哈哈。突然发现其实 < 3 和 < 2也没区别，真的要限制也应该是给个: 大于3, 然后小于多少次数(避免给老伙计推送了)，我就感觉怎么记得我们那边给一次贡献的也推送了邀请码。

---

> 要不条件改宽一点，过去有很多人贡献过 antd。

这个pr的初衷是简化评论区，2、3好像区别不大，感觉可以设为5（5次 commit 应该需要2次 pr 来完成）🤗

---

hhh, 因为squash merge 实际上一个pr算进去只有一个commit好像，门槛太高可能对新人门槛有点高，看大佬们选择多少次了，我上面是说因为现在都是给 < 某某次次数的人发，但是上面 建议的 < 3 次的代码。并没有提高门槛。 < 2 本身也<3了。` > 3 && < 5 `这样子也行。

---

我好像理解反了，好像就是为了给更多人推送是么，还以为要预防坏人进群呢，最近蹭pr的太多了 😅

---

嗯嗯，核心成员鼓励大家参与开源共建antd ，社区贡献者有2k+ ，所以避免之前贡献过没有收到评论，多评论总归是好的。

---

> > 要不条件改宽一点，过去有很多人贡献过 antd。
> 
> 这个pr的初衷是简化评论区，2、3好像区别不大，感觉可以设为5（5次 commit 应该需要2次 pr 来完成）🤗

merged，明早可以拿着这个pr加群了